### PR TITLE
Remove pips

### PR DESCRIPTION
**Pull request by evgeniysergeev**
There no such thing in original

### CODE REVIEW

**Comment by MustaphaTR** [comment]
There original showed the percentage for the harvester when you select it via text and silo storage on the sidebar when you select a silo or refinery tho. I'm not sure if it is a good idea to remove the pip without having this first.

**Comment by evgeniysergeev** [comment]
Waiting #124

**Comment by Phrohdoh** [comment]
Can this be revisited now that #124 is closed?

**Comment by abcdefg30** [comment]
Yes I suppose so. Harvesters however don't seem to display this information yet @evgeniysergeev.

**Comment by evgeniysergeev** [comment]
> Yes I suppose so. Harvesters however don't seem to display this information yet @evgeniysergeev.

Harvester show information not on the left side(which is ready) but on top bar.
But unfortunately it is hard to draw this top bar using original resources. OpenRA engine can handle only png files for UI, but original resources is in SCP files.
So currently don’t know what to do with this

**Comment by abcdefg30** [comment]
You could either try to convert the required SCP parts to PNG or add a text (like for refineries) as workaround until you can use the SCP files.

**Comment by TheRaffy** [comment]
SCP? is that a file ending?

**Comment by evgeniysergeev** [comment]
> 
> 
> SCP? is that a file ending?

CPS:

SCREEN_CPS

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1 @@
+../d2/mods/d2/AUTHORS
```
Comment (Phrohdoh) [inline AUTHORS]: This path is funky.
I _suspect_ you want `./mods/d2/AUTHORS` if you're going to put a path in this file.

**Comment by abcdefg30** [comment]
Looking at the odd commit display, this PR probably needs a rebase?

**Comment by evgeniysergeev** [comment]
> 
> 
> Looking at the odd commit display, this PR probably needs a rebase?

It's hard to maintain this pull request. so Created #148 instead. Closing this PR for now. It will be made after Top bar will be drawn together with sidebar

**Pull request closed without merging**